content/docs/command-reference/repro.md

```diff
@@ -93,6 +96,25 @@ up-to-date and only execute the final stage.
 
 ## Options
 
+- `<target>` - Specify the stages to reproduce.
+
+  Target can be a name of the stage in the dvc.yaml file or a path to a .dvc or
+  a dvc.yaml file. In case of a file, it will reproduce all of the stages
+  present in that file.
+
+  With `-R`, the target can be a directory to search for stages.
+
+  With `--glob`, the targets are used as a pattern to match stages in the
+  dvc.yaml file (example: `train-*`).
+
+  A stage from a dvc.yaml in a different directory can be specified using a path
```

Comment:
I wonder if in this case it's easier to just show 5 or more examples with their meaning (instead of explaining `:` explicitly)?

Reply:
Yeah, it might be too much to read if you just need to get it done. But, I think it's good to also layout the _spec_ of it, so I added an example for each scenario.

Reply:
yeah, spec is good, and I saw your examples (that's why I'm actually suggesting to do emphasis on them) the question is - should the spec be part of the examples explanation? this way most people with get the idea from what they see, almost immediately, and they can still read if needed.

Reply:
That sounds good. Will make changes later in the day. Thanks.

Reply:
> > easier to just show 5 or more examples with their meaning (instead of explaining :
>
> it might be too much to read if you just need to get it done

It's good to have the spec thoough. I can probably help summarize this text when it's ready. Consider instead adding an actual **Example** about wildcards in addition to this (and link to it from here).